poly/pvss.go

```diff
@@ -0,0 +1,298 @@
+package poly
+
+import (
+	"errors"
+
+	"github.com/dedis/cothority/crypto"
+	"github.com/dedis/crypto/abstract"
+	"github.com/dedis/crypto/random"
+)
+
+// This package implements public verifiable secret sharing (PVSS) using
+// non-interactive zero-knowledge (NIZK) proofs to show equality of discrete
+// logarithms (DLE).
```

Comment:
If you want the documentation to display nicely, you could either add a `doc.go` file with a concise explanation or (if you are short in time, which I guess never happens ;-) you could simply move this comment to the top. Then it would need to start with `// Package poly ...`. See: https://blog.golang.org/godoc-documenting-go-code.
Also, I think a link to (or at least name of) the paper where the exact algo is described would be useful.